guides/plugins/plugins/testing/end-to-end-testing.md

```diff
@@ -452,39 +470,20 @@ return Promise.all([
+That's it! There you go: You have successfully created a customised service that sets up a shipping method in Shopware. Actually, we use this service in our platform test to create our shipping method as well. You can find the full service [here](https://github.com/shopware/e2e-testsuite-platform/blob/master/cypress/support/service/administration/fixture/shipping.fixture.js). So please look at this example to see the whole class.
 
 Below you will find some best practices and tricks we explored to help you with your testing tasks:
 
-* A source of information can be found in FieldCollection of the several EntityDefinition files.
-
-  All fields belonging to an entity are defined there. For example, if you're searching for customer related data,
-
-  please search for the `CustomerDefinition` in Shopware platform.
-
-* Furthermore, you can always look at our Swagger UI every Shopware installation is coming with to
-
-  find out the correct data structure besides looking up the EntityDefinitions. The UI can be found via
-
-  `http://<your-installation>/api/v3/_info/swagger.html`.
-
-* If you want to extract mandatory data that is not covered by the error message received with the API's response,
-
-  it's useful to reproduce your workflow manually: E.g. if you need to find out what data is mandatory for creating
-
-  a customer, try to save an empty one in the administration. Keep an eye on the developer tools of your browser while
-
-  doing so, especially on the preview and response section of your request. As you get your response,
-
-  you can see what data is still missing.
-
-* If you need to set non-mandatory data, reproducing the above mentioned workflow is recommended as well:
-
-  Even if the error response does not contain a readable error, you can still inspect it: All the relevant information
-
-  is stored in 'data'. IDs can be found there directly, other relevant data is stored in "attributes".
-
-* Cypress' test runner can help you a lot with inspecting API requests. Just click on the request in the test runner's
+* A source of information can be found in FieldCollection of the several EntityDefinition files. All fields belonging to an entity are defined there. For example, if you're searching for customer related data, please search for the `CustomerDefinition` in Shopware platform.
+* Furthermore, you can always look at our Swagger UI every Shopware installation is coming with to find out the correct data structure besides looking up the EntityDefinitions. The UI can be found via`http://<your-installation>/api/v3/_info/swagger.html`.
```

Comment:
just a question - with 6.5 we get rid of this api-versioning afaik . Can you check that?

Reply:
Possible, I'd remove the bullet point. I think that's better, isn't it?

Reply:
I think its okay!